scanner/matcher/matcher.go

```diff
@@ -40,43 +42,68 @@ func NewMatcher(ctx context.Context, cfg config.MatcherConfig) (Matcher, error)
 	if err != nil {
 		return nil, fmt.Errorf("initializing postgres matcher store: %w", err)
 	}
+	metadataStore, err := metadatapostgres.InitPostgresMetadataStore(ctx, pool, true)
+	if err != nil {
+		return nil, fmt.Errorf("initializing postgres matcher metadata store: %w", err)
+	}
 	locker, err := ctxlock.New(ctx, pool)
 	if err != nil {
 		return nil, fmt.Errorf("creating matcher postgres locker: %w", err)
 	}
-	// TODO: Remove when Scanner V4 updater pipeline is available.
-	if err := updaterdefaults.Error(); err != nil {
-		return nil, fmt.Errorf("vulnerability updater init: %w", err)
-	}
+
 	// TODO: Update HTTP client.
 	c := http.DefaultClient
+
 	libVuln, err := libvuln.New(ctx, &libvuln.Options{
-		Store:  store,
-		Locker: locker,
-		// TODO: Enable when Scanner V4 updater pipeline is available.
-		// // Run in "air-gapped" mode.
-		// DisableBackgroundUpdates: true,
-		UpdateRetention: libvuln.DefaultUpdateRetention,
-		Client:          c,
-		Enrichers: []driver.Enricher{
-			&cvss.Enricher{},
-		},
+		Store:                    store,
+		Locker:                   locker,
+		DisableBackgroundUpdates: true,
+		UpdateRetention:          libvuln.DefaultUpdateRetention,
+		Client:                   c,
+		// TODO: Replace with our own enrichers.
+		Enrichers: nil,
 	})
 	if err != nil {
 		return nil, fmt.Errorf("creating libvuln: %w", err)
 	}
+
+	u, err := updater.New(updater.Opts{
+		Store:         store,
+		Locker:        locker,
+		Pool:          pool,
+		MetadataStore: metadataStore,
+		Client:        c,
+		// TODO: temporary URL
+		URL: "https://storage.googleapis.com/scanner-v4-test/vulnerability-bundles/4.3.x-173-g6bbb2e07dc/output.json.zst",
+	})
+	if err != nil {
+		return nil, fmt.Errorf("creating vuln updater: %w", err)
+	}
+
+	go func() {
```

Comment:
This allows the matcher to be used even without vulnerabilities. We need to decide how to determine when the matcher is `ready`